helmsman latest release is not compatible with helm 3.3.2

Helm 3.3.2 introduced [a breaking change](https://github.com/helm/helm/commit/882eeac7271858124a3cecbe22c5d7d61560714f) (on a patch release), helm repo add will fail if the repo has already been added.
This is something that happens often with Helmsman, running the tests on master with this version of Helm now fails:

```
❯ make build test
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /Users/ndegory/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /Users/ndegory/.kube/config
Error: repository name (stable) already exists, please specify a different name
make: *** [repo] Error 1
```

--- a/internal/app/cli.go
+++ b/internal/app/cli.go
@@ -6,7 +6,6 @@ import (
 	"os"
 	"strings"
 
-	version "github.com/hashicorp/go-version"
 	"github.com/imdario/mergo"
 	"github.com/joho/godotenv"
 )
@@ -144,15 +143,8 @@ func (c *cli) parse() {
 		c.parallel = 1
 	}
 
-	helmVersion := strings.TrimSpace(getHelmVersion())
-	extractedHelmVersion := helmVersion
-	if !strings.HasPrefix(helmVersion, "v") {
-		extractedHelmVersion = strings.TrimSpace(strings.Split(helmVersion, ":")[1])
-	}
-	log.Verbose("Helm client version: " + extractedHelmVersion)
-	v1, _ := version.NewVersion(extractedHelmVersion)
-	jsonConstraint, _ := version.NewConstraint(">=3.0.0")
-	if !jsonConstraint.Check(v1) {
+	log.Verbose("Helm client version: " + strings.TrimSpace(getHelmVersion()))
+	if checkHelmVersion("<3.0.0") {
 		log.Fatal("this version of Helmsman does not work with helm releases older than 3.0.0")
 	}
 

--- a/internal/app/helm_helpers.go
+++ b/internal/app/helm_helpers.go
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/go-version"
 	"net/url"
 	"strings"
 
@@ -55,6 +56,20 @@ func getHelmVersion() string {
 	}
 
 	return result.output
+}
+
+func checkHelmVersion(constraint string) bool {
+	helmVersion := strings.TrimSpace(getHelmVersion())
+	extractedHelmVersion := helmVersion
+	if !strings.HasPrefix(helmVersion, "v") {
+		extractedHelmVersion = strings.TrimSpace(strings.Split(helmVersion, ":")[1])
+	}
+	v, _ := version.NewVersion(extractedHelmVersion)
+	jsonConstraint, _ := version.NewConstraint(constraint)
+	if jsonConstraint.Check(v) {
+		return true
+	}
+	return false
 }
 
 // helmPluginExists returns true if the plugin is present in the environment and false otherwise.
@@ -132,7 +147,11 @@ func addHelmRepos(repos map[string]string) error {
 			repoLink = u.String()
 		}
 
-		cmd := helmCmd(concat([]string{"repo", "add", repoName, repoLink}, basicAuthArgs), "Adding helm repository [ "+repoName+" ]")
+		repoAddFlags := ""
+		if checkHelmVersion(">=3.3.2") {
+			repoAddFlags += "--force-update"
+		}
+		cmd := helmCmd(concat([]string{"repo", "add", repoAddFlags, repoName, repoLink}, basicAuthArgs), "Adding helm repository [ "+repoName+" ]")
 		// check current repository against existing repositories map in order to make sure it's missing and needs to be added
 		if existingRepoUrl, ok := existingRepos[repoName]; ok {
 			if repoLink == existingRepoUrl {

